### PR TITLE
NCL-7231: Set originalVersion in subprojects

### PR DESCRIPTION
**Pull request by dwalluck**

### CODE REVIEW

File: manipulation/src/main/java/org/jboss/gm/manipulation/actions/ManifestUpdateAction.java [hunk 1]
```diff
@@ -94,10 +94,15 @@ public void execute(Project project) {
...
-                    logger.warn("For task {}, overriding {} value {} with {}", j.getName(), e,
-                            manifest.getAttributes().get(e), alignmentModel.getVersion());
+                    Object value = manifest.getAttributes().get(e);
+                    if (!value.toString().equals(alignmentModel.getVersion())) {
+                        logger.warn("For task {}, overriding {} value {} with {}", j.getName(), e, value,
+                                alignmentModel.getVersion());
+                        manifest.getAttributes().put(e, alignmentModel.getVersion());
+                    }
+                } else {
+                    manifest.getAttributes().put(e, alignmentModel.getVersion());
```
Comment (rnc) [inline manipulation/src/main/java/org/jboss/gm/manipulation/actions/ManifestUpdateAction.java:104]: This block change seems separate to the aim of the PR (`Set originalVersion in subprojects`). Was the manifest not being set in certain circumstances? Could this be verified by a test? Perhaps this would be better factored out into a separate PR ?

Comment (dwalluck) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/ManifestUpdateAction.java:104]: OK, I will take it out of this PR. It's related in that it needs the first part in order to work.

Comment (dwalluck) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/ManifestUpdateAction.java]: Removed in [43301c1](https://github.com/project-ncl/gradle-manipulator/commit/43301c1faa40d170d959d1cf4fddb34c073aeff8).

**Review by rnc — APPROVED** [review]
